tests/selection.spec.ts

```diff
@@ -38,23 +38,23 @@ test('click on blank area', async ({ page }) => {
   const above123 = await page.evaluate(() => {
     const paragraph = document.querySelector('[data-block-id="2"] p');
     const bbox = paragraph?.getBoundingClientRect() as DOMRect;
-    return { x: bbox.left, y: bbox.top - 5 };
+    return { x: bbox.left, y: bbox.top + 5 };
```

Comment:
The mouse operates is relative to the top-left corner of the viewport in the playwright. So the minus calculations caused the operator can not click into the block on Windows.

See https://playwright.dev/docs/api/class-mouse